app/src/docs/content/gettingStarted/gettingStarted.mdx

```diff
@@ -14,7 +15,15 @@ import SignatureRequest2x from './images/signatureRequest@2x.png'
 
 # Getting Started
 Welcome to Streamr! To get a feel for what Streamr is, check out our <Link to={docsLinks.introduction}>intro docs</Link>.
-The first thing you'll want to get is a Streamr account. You can do that <Link to={routes.auth.login()}>here</Link>. Creating an account requires either an Email address or a Web3 Ethereum wallet, e.g. <a href="https://metamask.io" target="blank">Metamask</a>.
+The first thing you'll want to get is a Streamr account. You can do that <Link to={routes.auth.login()}>here</Link>. Creating an account requires a Web3 Ethereum wallet, e.g. <a href="https://metamask.io" target="blank">Metamask</a>.
```

Comment:
> The first thing you'll want to get is a Streamr account.

Is there such a thing anymore as Streamr account if the user can just login with their Metamask?

Reply:
Not quite ready for this existential question! I think it's ok to talk about Streamr accounts for now. Not sure if it's more confusing or less confusing to talk about Eth accounts as the 'account' to use.

Reply:
In any case I would maybe change the wording so that the user can just login, no need to "get" an Account since the register function is gone - just login and your account is created.

Reply:
Ah, that's a good point.